hack/charts/installer/rolebased/values.yaml

```diff
@@ -2,5 +2,5 @@ roleArn: arn:aws:iam::123456789012:role/DELETE_ME
 
 image:
   repository: 957583890962.dkr.ecr.us-east-1.amazonaws.com/amazon-sagemaker-operator-for-k8s
-  tag: latest
```

Comment:
We need to change our current installer [yaml](https://github.com/aws/amazon-sagemaker-operator-for-k8s/blob/master/release/rolebased/installer.yaml#L4014) and [helm chart](https://github.com/aws/amazon-sagemaker-operator-for-k8s/blob/master/hack/charts/installer/rolebased/values.yaml#L5).